### DISCUSSION TITLE
Post form to REST-API

### DISCUSSION THREAD

**Discussion by mnn-softjoy** [Budibase/budibase#4326] [Help]
Hi!

I have tried the film variants and now I need to reach out for help.

I have a REST POST api like http://URL:port/apiv1/account/1. The idea is to update and create accounts. In BB i have created a form with one submit button variant "primary". 
![image](https://user-images.githubusercontent.com/98225460/152555891-cf0a6287-fb64-40f2-85d4-ec3b4419a53b.png)
On that I have one action to run query.

I have tried Bindings. 
![image](https://user-images.githubusercontent.com/98225460/152555704-eca22322-1275-4618-9077-c186bcabf134.png)
 
The help only gives me the string "{{ New Form.Fields. }}". It's not like the film.

The json body looks like this
![image](https://user-images.githubusercontent.com/98225460/152557249-b770c8dc-1411-49d8-ac9d-35cdf8cca6f9.png)

The id is updated, The name gets the default value

**Comment by mjashanks** [comment]
I'm sorry - I don't understand your issue. Is this not working in some way? What happens when your query gets executed.

Thanks

**Reply by mnn-softjoy** [reply]
Ah. The REST interface gets the default value for name and not the value in the form
